src/Settings/Builders/Ghc.hs

```diff
@@ -79,6 +79,7 @@ commonGhcArgs = do
             , arg "-hcsuf", arg $ hcsuf way
             , wayGhcArgs
             , packageGhcArgs
+            , arg "-lnuma"
```

Comment:
@snowleopard I incurred an error which I think is due to #559 and added an additional flag to resolve this. I accidentally committed the change but think might be helpful. Should I remove the argument?

Reply:
We shouldn't add such flags unconditionally. Better do this in a separate PR.

Reply:
Resolved